src/client/components/dragAndDrop/drag-and-drop.css

```diff
@@ -0,0 +1,24 @@
+.container:hover {
+    color: rgb(255, 255, 255);
```

Comment:
Don't use RGB, use please standard form of color (#*******)

Reply:
done